Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Squad/prep.yml

```
@@ -160,8 +160,8 @@
       entries:
       - id: CMBackpackWelder
         amount: 1
-#      - id: Technician Welder-Satchel
-#        amount: 2
+      - id: Technician Welder-Satchel
```

I think you meant to put the entity ID here?
```suggestion
      - id: RMCSatchelWelder
```

---

oh god.